### DISCUSSION TITLE
在云服务器上本地启动成功，怎么设置外网ip访问

### DISCUSSION THREAD

**Discussion by hulukaihuaba** [babaohuang/GeminiProChat#17] [Q&A]
![微信图片_20231218114829](https://github.com/babaohuang/GeminiProChat/assets/9246009/a8948bdf-2085-4e13-9f80-cf6659e743f7)
需要怎么操作，设置外网访问？

**Comment by dmych1208** [comment]
这是哪个文件？

**Comment by bignewnew** [comment]
我也遇到了这个问题，如果在云服务器上，是不能通过npm run dev 访问的吧~

**Comment by bignewnew** [comment]
使用宝塔面板，在“网站”-“node”项目中进行部署即可。不需要npm run build，直接自动获取package.json文件中的启动模式即可
<img width="1275" alt="image" src="https://github.com/babaohuang/GeminiProChat/assets/111179733/6cbdcc49-78f9-4176-8676-3fc045f04203">
